docs/getting-started/summaryofchanges.md

```diff
@@ -96,6 +96,22 @@ Review the [Zowe Explorer Change Log](https://github.com/zowe/vscode-extension-f
 
 You can install the latest version of the extension from the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe).
 
+### Zowe installer
+
+- A `-l` parameter that controls where the setup scripts write trace logs is added to `zowe-install.sh`, `zowe-configure-instance.sh`, `zowe-create-keystore.sh`, `zowe-install-xmem.sh` and `zowe-install-proc.sh`.  The problem determination gathering script `zowe-support.sh` has a `-l` parameter to specify the locaiton of where the scripts have been collected.  
```

Comment:
`-l` is not on zowe-configure-instance.sh. There is no such thing as `zowe-create-keystore.sh` - I assume you mean `zowe-setup-certificates.sh`? zowe-support.sh's -l paramter didn't make it into 1.11

Reply:
My bad @stevenhorsman .  I've fixed this and also removed the `zowe-install.sh` reference as I think that was in 1.10 already, and I've added doc links to the relevent sections to where each script is described

Reply:
`-l` wasn't in `zowe-install.sh` in 1.10, so it should be added back in!